src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Shape;
+import core.basesyntax.figures.Square;
+import core.basesyntax.figures.Triangle;
+import java.util.Random;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Random random = new Random();
+        Shape[] figures = new Shape[random.nextInt(10) + 1];
+
+        for (int i = 0; i < figures.length; i++) {
+            switch (random.nextInt(6) + 1) {
```

Comment:
6 is a magic number)
let's create a constant for it - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names